symfony/plugins/orangehrmAttendancePlugin/Dao/AttendanceDao.php

```diff
@@ -626,4 +628,89 @@ public function getLatestAttendanceRecordByEmployeeNumber(int $employeeNumber):
 
         return $q->getQuery()->getOneOrNullResult();
     }
+
+    /**
+     * @param AttendanceReportSearchFilterParams $attendanceReportSearchFilterParams
+     * @return array
```

Comment:
you can add key list of associate array, or example

Reply:
done